src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/driver")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars/drivers/add")
```
let's use naming that is suggested in the checklist.
and don't forget to change this link in jsp as well